src/components/flash-messages/FlashMessage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const TYPE = {
+  DEFAULT: 'default',
+  SUCCESS: 'success',
+  ERROR: 'error',
+  INFO: 'info'
+};
+
+const FlashMessage = ({text, type = TYPE.DEFAULT}) => {
+  const messageClass = classNames('sg-flash__message', {
+    [`sg-flash__message--${type}`]: type !== TYPE.DEFAULT
+  });
+
+  return <div className="sg-flash">
+    <div className={messageClass}>
+      <div className="sg-text sg-text--emphasised sg-text--small sg-text--light">
```

Comment:
Shouldn't we do `<Text>` before doing this? Or are you planning to come back to these components and fix it when `<Text>` is ready? Or ,for some reason, we are skipping `<Text>`?